src/flask_config.py

```diff
@@ -5,3 +5,5 @@ class FlaskConfig:
     def __init__(self):
         self.secret_key = os.environ.get('FLASK_SECRET_KEY').strip()
         self.login_disabled = os.environ.get('FLASK_LOGIN_DISABLED', False)
+        self.log_level = os.environ.get('FLASK_LOG_LEVEL', 'ERROR')
```

Comment:
Since you haven't added FLASK_LOG_LEVEL to your terraform, production is defaulting to a log level of ERROR?

You probably want INFO.